controller/pod.go

```diff
@@ -283,6 +288,86 @@ func parsePodConfigFromAnnotationsKuma(annotations map[string]string, pod *podCo
 	}
 }
 
+func parsePodConfigFromAnnotationsOsm(annotations map[string]string, pod *podConfig) {
+	statusPort := 15021
+	if v, ok := annotations["openservicemesh.io/port"]; ok {
+		vv, err := strconv.ParseUint(v, 10, 16)
+		if err == nil {
+			statusPort = int(vv)
+		}
+	}
+	pod.statusPort = uint16(statusPort)
+	excludeInboundPorts := []uint16{15000, 15001, 15003, 15010, 15021, 15050, 15128, 15901, 15902, 15903, 15904}
```

Comment:
Currently, `MaxItemLen` is 10, maybe can not store too many ports.

Increase the `MaxItemLen` if it necessary.

Reply:
BTW I think 15021 is overlapped with `statusPort`, maybe we should merge them in the future